src/include/ceph_fs.h

```diff
@@ -739,6 +739,7 @@ int ceph_flags_to_mode(int flags);
 				 CEPH_CAP_XATTR_SHARED)
 #define CEPH_STAT_CAP_INLINE_DATA (CEPH_CAP_FILE_SHARED | \
 				   CEPH_CAP_FILE_RD)
+#define CEPH_STAT_RSTAT        (CEPH_CAP_GWREXTEND << 16)  /* for requesting rstat */
```

Comment:
Oops, CEPH_STAT_RSTAT is (1 << 22).  I assumed it's CEPH_CAP_FILE_WREXTEND

Reply:
emm, CEPH_CAP_FILE_WREXTEND is (64 << 8). CEPH_STAT_RSTAT is (64 << 16). I think no problem is here.

Reply:
I changed it to CEPH_CAP_FILE_WREXTEND

https://github.com/ceph/ceph/pull/21633/commits/ca6c8a7a1956691837948c38ff7c5b7c45f2a051

Reply:
Ok, got it, thanks for changing it to a proper value. 

BTW, I have seen your changes on kernel cephfs and Jeff's review. Pls let me know if flag parameter on ceph-fuse also needs to be unsigned, then I can make the changes. :)

Reply:
please do it if you have time